ec2/sc-ec2-linux-jumpcloud.yaml

```diff
@@ -355,6 +349,45 @@ Resources:
             -
               yum:
                 jq: []
+          files:
+            /opt/sage/bin/make_env_vars_file.sh:
+              content: !Sub |
+                #! /bin/bash
+
+                EC2_INSTANCE_ID=$(/usr/bin/curl -s http://169.254.169.254/latest/meta-data/instance-id)
+                ROOT_DISK_ID=$(/usr/bin/aws ec2 describe-volumes \
+                  --region $AWS_REGION \
+                  --filters Name=attachment.instance-id,Values=$EC2_INSTANCE_ID \
+                  --query Volumes[].VolumeId --out text)
+                EC2_INSTANCE_TAGS=$(aws --region $AWS_REGION \
+                  ec2 describe-tags \
+                  --filters Name=resource-id,Values=$EC2_INSTANCE_ID)
+                extract_tag_value() {
+                  echo $EC2_INSTANCE_TAGS | jq -j --arg KEYNAME "$1" '.Tags[] | select(.Key == $KEYNAME).Value '
+                }
+                DEPARTMENT=$(extract_tag_value Department)
+                PROJECT=$(extract_tag_value Project)
+                OWNER_EMAIL=$(extract_tag_value OwnerEmail)
+
+                echo "export AWS_REGION=$AWS_REGION" >> /opt/sage/bin/export_env_vars.sh
```

Comment:
export_env_vars.sh doesn't exist yet, so you may want to either create it first or use `>` instead?  

Hmm, also i'm wondering what will happen if this gets run again? because this keeps appending to the file.

Reply:
`>>` creates a file if it does not exist yet.
Yes, if this gets run again it keeps appending to file.